core/templates/pages/android-page/android-page.component.css

```diff
@@ -290,9 +290,24 @@
   color: #fff;
   padding: 15px 20px;
   position: relative;
+  transition: background-color 0.3s ease;
   z-index: 100;
 }
 
+.oppia-android-carousel-chevron:hover {
+  background-color: #3d9991;
+}
+
+.oppia-android-carousel-button {
+  background: none;
+  border: none;
+}
+
+.oppia-android-carousel-button:active,
+.oppia-android-carousel-button:focus {
+  outline: none; 
```

Comment:
Please remove the terminating white spaces (if any). I think this is the cause of lint failure.

Reply:
okay